devicemanager.go

```diff
@@ -97,11 +103,18 @@ func (dm *DeviceManager) renewLoop() {
 			logger.Info.Printf("Renewing lease for device:%s\n", dm.Name())
 			if err := dm.renewLease(); err != nil {
 				logger.Error.Printf("Cannot update lease, will retry in one sec: %s", err)
```

Comment:
```suggestion
				logger.Error.Printf("Cannot update lease, will retry in %s: %s", dm.backoff.Duration(), err)
```

Update the log statement to include the backoff duration?

Reply:
hmm that would jump in step every time as Duration() increments the attempts. Let me think about it and I'll update.

Reply:
updated